scripts/shared/lib/deploy_helm

```diff
@@ -54,31 +54,37 @@ function helm_install_subm() {
         return
     fi
 
+    flags=(   --kube-context "${cluster}" \
+              --name submariner \
+              --namespace "${SUBM_NS}" \
+              --set "ipsec.psk=${SUBMARINER_PSK}" \
+              --set "broker.server=${submariner_broker_url}" \
+              --set "broker.token=${submariner_broker_token}" \
+              --set "broker.namespace=${SUBMARINER_BROKER_NS}" \
+              --set "broker.ca=${submariner_broker_ca}" \
+              --set "submariner.clusterId=${cluster}" \
+              --set "submariner.clusterCidr=${cluster_CIDRs[$cluster]}" \
+              --set "submariner.serviceCidr=${service_CIDRs[$cluster]}" \
+              --set "submariner.globalCidr=${global_CIDRs[$cluster]}" \
+              --set "serviceAccounts.globalnet.create=${globalnet}" \
+              --set "submariner.natEnabled=false" \
+              --set "routeAgent.image.repository=localhost:5000/submariner-route-agent" \
+              --set "routeAgent.image.tag=local" \
+              --set "routeAgent.image.pullPolicy=IfNotPresent" \
+              --set "engine.image.repository=localhost:5000/submariner" \
+              --set "engine.image.tag=local" \
+              --set "engine.image.pullPolicy=IfNotPresent" \
+              --set "globalnet.image.repository=localhost:5000/submariner-globalnet" \
+              --set "globalnet.image.tag=local" \
+              --set "globalnet.image.pullPolicy=IfNotPresent" \
+              --set "crd.create=${crd_create}" )
+
+    if [[ "${cableDriver}" != "default" ]] ; then
```

Comment:
Since `default` is a fake value, why not just send nothing in case it's not specified and here just check if the value is not empty?